pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -388,10 +388,9 @@ func (pl *InterPodAffinity) RemovePod(ctx context.Context, cycleState *framework
 func getPreFilterState(cycleState *framework.CycleState) (*preFilterState, error) {
 	c, err := cycleState.Read(preFilterStateKey)
 	if err != nil {
-		// The preFilterState wasn't pre-computed in prefilter. We ignore the error for now since
-		// Filter is able to handle that by computing it again.
+		// The preFilterState wasn't pre-computed in prefilter.
 		klog.V(5).Infof("Error reading %q from cycleState: %v", preFilterStateKey, err)
-		return nil, nil
+		return nil, err
```

Comment:
```suggestion
		return nil, fmt.Errorf("error reading %q from cycleState: %v", preFilterStateKey, err)
```

And remove the above line.

Reply:
Thanks @Huang-Wei, done.